lib/alephant/preview/server.rb

```diff
@@ -49,7 +49,7 @@ class Server < Sinatra::Base
         batch_components = []
 
         batched_components.each do |component|
-          options = component.fetch(:options, {}) || {}
+          options = symbolize(component.fetch(:options, {}) || {})
```

Comment:
Hi @samfrench I checked this locally and I don't think you need the final OR operator...

```rb
h = {"components"=>[{"component"=>"ni_constituency_title","options"=>[]}]}
component = h["components"][0]
component.fetch("options", {}) || {} # => []
```

Notice that last line still gives back an empty Array and doesn't default to giving back an empty hash

So you might as well just use `fetch` and if an empty Array is found `[]` then pass that to symbolize...

```rb
options = symbolize component.fetch("options", {})
```